evansmusomi/react-native-102
# Styles Share Place Screen and Extracts Components

*What this does*
- Styles share place screen
- Extracts a PickImage, PlaceInput and PickLocation component

*How to verify*
- Elements on the share place screen should be centered and styled: the image preview and picker, the map preview and location picker, the place input and share place button
- There should be PickImage, PickLocation, PlaceInput components in the components folder
